src/main/scala/io/viash/Main.scala

```diff
@@ -45,6 +45,8 @@ object Main {
   val version: String = if (pkg.getImplementationVersion != null) pkg.getImplementationVersion else "test"
 
   val viashHome = Paths.get(sys.env.getOrElse("VIASH_HOME", sys.env("HOME") + "/.viash"))
```

Comment:
does this also need to be updated?

Reply:
Will be handled in a separate PR where I plan to wrap all sys.env calls so we can track and document them